icepyx/core/read.py

```diff
@@ -628,7 +628,11 @@ def load(self):
                 # If path is an s3 path create an s3fs filesystem to reference the file
                 # TODO would it be better to be able to generate an s3fs session from the Mixin?
                 s3 = earthaccess.get_s3fs_session(daac="NSIDC")
-                file = s3.open(file, "rb")
+                fsspec_params = {
+                    "cache_type": "blockcache",
+                    "block_size": 8 * 1024 * 1024,
+                }
+                file = s3.open(file, "rb", **fsspec_params)
```

Comment:
I'm thinking if there's a way to just call [`earthaccess.open`](https://earthaccess.readthedocs.io/en/latest/user-reference/api/api/#earthaccess.api.open) here? We want to slowly delegate most of the granule reading logic to earthaccess (xref https://github.com/icesat2py/icepyx/issues/575), so would be best if we can have `earthaccess` handle the default cache/block sizes somehow. Cc @betolink for thoughts (since you had some ideas at https://github.com/nsidc/earthaccess/discussions/527).

Reply:
>We want to slowly delegate most of the granule reading logic to earthaccess (xref https://github.com/icesat2py/icepyx/issues/575), so would be best if we can have earthaccess handle the default cache/block sizes somehow.

Totally agreed - but I think optimized fsspec parameters (or users setting them in the open call) isn't yet implemented in earthaccess, right? If that's the case, maybe we should open an issue within icepyx that tracks the work in earthaccess so that once it's implemented we can update these lines in the Read module, but in the meantime we can take advantage of the faster read speeds made possible by setting the fsspec parameters. Thoughts?